lib/commits.js

```diff
@@ -12,7 +12,7 @@ module.exports = tags =>
     const [release, parent] = tags
 
     if (!release || !parent || !parent.hash || !release.hash) {
-      reject(new Error('the first release should be created manually.'))
+      reject(new Error('the first two releases should be created manually.'))
```

Comment:
Yes, please do! But I get your point. But with the current solution, people will think that TWO actual github releases need to be created, which is not true. Only two tags... 🤔